tests/WE2E/test_configs/custom_grids/config.custom_ESGgrid_Great_Lakes_snow_8km.yaml

```diff
@@ -2,8 +2,8 @@ metadata:
   description: |-
     This test checks the capability of the workflow to run verification tasks
     on a user-specified grid. This also serves as a winter snowstorm case for
-    checking snowfall verification statistics, and a test for custom domains
-    with staged netCDF data.
+    checking snowfall verification statistics using observations retrieved from
+    HPSS, and a test for custom domains with RAP data retrieved from HPSS
 user:
```

Comment:
Could HRRR ICs be used for this domain too?  Just curious why RAP is mentioned by itself.

Reply:
I was simply making the description more verbose: this test specifically retrieves RAP data, but GFS and HRRR (and GEFS/GDAS for that matter) can also be retrieved.

Now that you mention it though I'm pretty sure this custom domain might extend a bit north of the HRRR domain, so those ICs might not be possible for this specific domain.